Library/Homebrew/tap.rb

```diff
@@ -82,6 +82,16 @@ def git?
     (path/".git").exist?
   end
 
+  # The issues URL of this {Tap}.
+  # e.g. `https://github.com/user/homebrew-repo/issues`
+  def issues_url
+    if official?
+      "https://github.com/Homebrew/homebrew-#{repo.strip_prefix("homebrew-")}/issues"
+    else
+      "https://github.com/#{name}/issues"
```

Comment:
Good idea; I just did that. I check `official?` just in case it’s a custom remote for an official repo.